### ISSUE TITLE
dotnet --info hitting /lib64/libstdc++.so.6: version `GLIBCXX_3.4.20' not found on Latest net8.0

### ISSUE THREAD

**Issue by LoopedBard3** [dotnet/sdk#31893]
### Describe the bug
After installing the latest 8.0 daily dotnet using the dotnet-install script onto a clean ubuntu-20.04 or ubuntu-22.04 image, running dotnet --info errors out with:
```
dotnet: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.20' not found (required by dotnet)
dotnet: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.21' not found (required by dotnet)
```

### To Reproduce
On a clean ubuntu-22.04 image, download the dotnet-install.sh script.
Run: `.\dotnet-install.sh -Architecture x64 -Channel 8.0 -Quality daily`,
Run: `dotnet --info`
The failure should occur here. 

### Exceptions (if any)
```
dotnet: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.20' not found (required by dotnet)
dotnet: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.21' not found (required by dotnet)
```

### Further technical details
Last known working SDK version: 8.0.100-preview.3.23178.7
Failing SDK version: 8.0.100-preview.4.23219.2

Thank you!

**Comment by MichalStrehovsky** [comment]
@sbomer does this sound familiar?

**Comment by LoopedBard3** [comment]
I will take a look!

**Comment by LoopedBard3** [comment]
Looks like this was being hit because the centOS container we were using doesn't have the GLIBCXX versions in the default paths. I am not sure if they have them installed in general, but I found this on SO https://stackoverflow.com/questions/44205687/glibcxx-3-4-21-not-found-on-centos-7 that somewhat indicates it does. To fix this we switched the containers we could to ubuntu 18.04 containers. Thanks for the help!
